src/objects/core/zcl_abapgit_filename_logic.clas.abap

```diff
@@ -370,16 +370,23 @@ CLASS zcl_abapgit_filename_logic IMPLEMENTATION.
 
 
   METHOD object_to_i18n_file.
-    DATA: lv_langu_sap1 TYPE sy-langu,
+    DATA: lv_langu_sap1  TYPE sy-langu,
           lv_langu_bcp47 TYPE string.
 
     lv_langu_sap1 = zcl_abapgit_convert=>language_sap2_to_sap1( to_upper( iv_lang ) ).
     lv_langu_bcp47 = zcl_abapgit_convert=>language_sap1_to_bcp47( lv_langu_sap1 ).
 
-    rv_filename = object_to_file(
-      is_item  = is_item
-      iv_extra = |i18n.{ lv_langu_bcp47 }|
-      iv_ext   = iv_ext ).
+    IF iv_ext = `properties`.
+      rv_filename = object_to_file(
+        is_item  = is_item
+        iv_extra = |texts.{ lv_langu_bcp47 }|
```

Comment:
Is this correct? [Here](https://github.com/SAP/abap-file-formats/blob/main/docs/properties.md), we specified that the file shall be named `<object_name>.<object_type>.i18n.<language>.properties`.

Texts is only for text elements of classes.